_maps/map_files/NorthStar/north_star.dmm

```diff
@@ -16637,6 +16637,16 @@
 	},
 /turf/open/floor/iron/smooth_large,
 /area/station/hallway/secondary/entry)
+"emV" = (
+/obj/structure/closet/secure_closet/brig/genpop,
```

Comment:
Why are you adding genpop closets? wait did you create a new branch from your previous PR #74924?

Reply:
Now looking back at that PR the changes are exactly the same

Reply:
I don't know why, but my GitHub shows all the changes I've done when I open a new pr.

Reply:
You need to create a new branch for every PR, you have reused the same branch from your last PR hence all changes made back then are showing up here. For this one the solution is pretty simple, just copy & paste the original northstar.dmm from a new branch to undo all changes to it.